scripts/src/plugin_runner/utils.py

```diff
@@ -9,9 +9,13 @@ def configure_parser(parser: ArgumentParser):
         help='Output directory to store folders with output files',
     )
     parser.add_argument(
-        'language',
+        TaskNamedArgs.LANGUAGE.value,
         help='Programming language of code submissions',
     )
+    parser.add_argument(
+        TaskNamedArgs.DISTANCE_LIMIT.value,
+        help='Max distance between two vertices inside one cluster',
+    )
```

Comment:
Do we actually want to consider distance limit as a positional argument for all runners? It is actually used only in `cluster` task

Reply:
Could you please fix it in another pr, since we don't have a python runner for this task yet?

Reply:
upd: I will redirect your PR to this branch